peachjam/models/citations.py

```diff
@@ -47,3 +47,20 @@ def from_extracted_citation(cls, citation):
                 },
             ],
         )
+
+
+class ExtractedCitation(models.Model):
+    citing_work = models.ForeignKey(
+        "peachjam.Work",
+        null=False,
+        on_delete=models.PROTECT,
```

Comment:
Cascade deletes, otherwise we can't easily delete the work.

For both foreign keys.